### PR TITLE
added gallery component

### PR DESCRIPTION
**Pull request by ReinisZogots**

### CODE REVIEW

File: src/components/molecules/gallery/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+interface GalleryData {
+  title: string,
+  subTitle: string,
+  galleryImages: GalleryImage[]
+}
+
+const Gallery: React.FC = () => {
+  const GalleryData: GalleryData = Structure.gallery;
+  const Images = GalleryData.galleryImages;
```
Comment (Jalal999) [inline src/components/molecules/gallery/index.tsx:18]: give type to Images

File: src/components/molecules/gallery/index.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Structure from "../../../structure.json";
+import "./style.scss";
+
+interface GalleryImage {
+  path: string,
+  altTag: string
+}
+
+interface GalleryData {
```
Comment (Jalal999) [inline src/components/molecules/gallery/index.tsx:10]: I think GalleryDataTypes would be better name

File: src/components/molecules/gallery/index.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  title: string,
+  subTitle: string,
+  galleryImages: GalleryImage[]
+}
+
+const Gallery: React.FC = () => {
```
Comment (Jalal999) [inline src/components/molecules/gallery/index.tsx:16]: I think, Gallery is big component for molecule. Take this component as an organism, divide it into parts and take its parts as molecules

**Review by ilze-dom — APPROVED** [review]

File: src/components/molecules/gallery/index.tsx (hunk 3 above)
Comment (ReinisZogots) [reply src/components/molecules/gallery/index.tsx]: split into smaller components

File: src/components/atoms/galleryImage/index.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+interface GalleryImagTypes {
```
[the remark this answers is not in the document]
Comment (ReinisZogots) [reply src/components/atoms/galleryImage/index.tsx:1]: changed for socialmediaicontypes as well

**Review by Jalal999 — APPROVED** [review]

**Pull request merged**